ext/openssl/openssl.c

```diff
@@ -4847,7 +4849,7 @@ PHP_FUNCTION(openssl_pkcs7_verify)
 
 	RETVAL_LONG(-1);
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "pl|pappp", &filename, &filename_len,
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "pl|p!a!p!p!p!", &filename, &filename_len,
```

Comment:
As far as I can see `$cainfo = []` should work.

Reply:
Though looking at other signatures, I see that this parameter is declared as `?array` in other cases, so this makes sense for consistency. Though the default is sometimes null and sometimes `[]`...

Reply:
You are right, so I'll change the default value of all of them to `[]`.